cypress/e2e/gui.cy.js

```diff
@@ -0,0 +1,163 @@
+/// <reference types = "cypress" />
+
+describe('EngageSphere gui test cases', () => {
+  beforeEach(() => {
+      cy.visit('/')
+      cy.get('button')
+      .contains('Accept')
+      .click()
```

Comment:
```suggestion
        .click()
```

Ajuste de indentação.